integration_tests/run_pyspark_from_build.sh

```diff
@@ -25,18 +25,10 @@ then
     >&2 echo "SPARK_HOME IS NOT SET CANNOT RUN PYTHON INTEGRATION TESTS..."
 else
     echo "WILL RUN TESTS WITH SPARK_HOME: ${SPARK_HOME}"
-    # support alternate local jars NOT building from the source code
```

Comment:
Why does this revert #1695?

Reply:
mistake when factoring this PR out of #1688, did not mean to include this file at all, will drop it